subprojects/android-test/test-instrumentation-runner/src/main/kotlin/com/avito/android/runner/InstrumentationTestRunner.kt

```diff
@@ -0,0 +1,32 @@
+package com.avito.android.runner
+
+import android.os.Bundle
+import androidx.test.runner.AndroidJUnitRunner
+
+abstract class InstrumentationTestRunner() : AndroidJUnitRunner() {
+
+    private var delegate: InstrumentationDelegate? = null
+
+    protected open fun createFactory(): ContextFactory {
+        return object : ContextFactory.Default() {
+            override fun createIfRealRun(arguments: Bundle): Context {
+                return DefaultTestInstrumentationContext(
+                    errorsReporter = LogErrorsReporter()
+                )
+            }
+        }
+    }
+
+    override fun onCreate(arguments: Bundle) {
+        val context = createFactory().create(arguments)
+        if (context != null) {
```

Comment:
Should we fail if null?

Reply:
It returns null when we in orchestrator execution